dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/GridHeader.java

```diff
@@ -67,6 +67,10 @@
 
     private LegendSet legendSet;
 
+    private String programStage;
+
+    private String stageOffset;
```

Comment:
Should this be Integer instead of String?

Reply:
In the GridHeader is stageOffset optional (valid only for repeatable stages and non default). I changed it to Optional<Integer>